### ISSUE TITLE
Networking: Add support for IPv6 configuration

### ISSUE THREAD

**Issue by mcastelino** [clearcontainers/runtime#593]
Currently our runtime lacks support for IPv6. We need to enhance the runtime, virtcontainers as well as our agent protocol and agent to support discovery and configuration of IPv6.

**Comment by egernst** [comment]
With the work going forward with Kata, I think we should close this issue as "will not fix" in CC, and perhaps track this in Kata?

**Comment by sboeuf** [comment]
@egernst I agree we won't implement this for Clear Containers. But we should open a duplicate of this issue on virtcontainers to track the progress.

**Comment by jodh-intel** [comment]
Kata issue is: https://github.com/kata-containers/runtime/issues/147.
